src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -960,9 +960,28 @@ class HookUtils(object):
         cgroup.assigned_resources = cgroup._get_assigned_cgroup_resources()
         # Create the cgroup(s) for the job
         cgroup.create_job(event.job.id, node)
+        if (cgroup.cfg['cgroup']['cpuset']['enabled']
+                and not cgroup.cfg['cgroup']['cpuset']['allow_zero_cpus']):
+            if ('ncpus' not in jobutil.assigned_resources
+                    or jobutil.assigned_resources['ncpus'] <= 0):
+                if event.job.in_ms_mom():
+                    pbs.logmsg(pbs.EVENT_ERROR,
+                               'cpuset enabled with mandatory ncpus >= 1, '
+                               'but job does not request ncpus on '
+                               'mother superior: rejecting job')
+                    event.reject('cpuset enabled with mandatory ncpus >= 1, '
+                                 'but job does not request ncpus on '
+                                 'mother superior: rejecting job')
+                else:
+                    pbs.logmsg(pbs.EVENT_JOB_USAGE,
+                               'cpuset enabled with mandatory ncpus >= 1, '
+                               'but job does not request ncpus on host: '
+                               'deleting cpuset cgroup')
+                # will be done in configure_job
```

Comment:
Comment should appear above pbs.logmsg() call.

Reply:
Corrected.